src/mapbox/mapbox.js

```diff
@@ -163,6 +163,15 @@ export default class Mapbox {
     // Reuse a saved map, if available
     if (props.reuseMaps && Mapbox.savedMap) {
       this._map = this.map = Mapbox.savedMap;
+      const oldContainer = this._map._container;
+      const newContainer = props.container;
+      // reparenting child nodes from old container to new container
```

Comment:
This is the meat

Reply:
Maybe move the comment up two lines and add a little more context about what is happening here. 

Is props.container a new prop. If so add to the propTypes and defaultProps above, and also update documentation.

Maybe for another PR, but I wonder if some unit test could be created for this file (even if it only runs in the browser). Just create one of these objects, finalize it, create another one, check the map is the same or different, depending on props...